src/Umbraco.Web.BackOffice/Controllers/BackOfficeController.cs

```diff
@@ -246,6 +251,33 @@ public async Task<JavaScriptResult> ServerVariables()
             return new JavaScriptResult(result);
         }
 
+        [HttpPost]
+        public ActionResult ExternalLogin(string provider, string redirectUrl = null)
+        {
+            if (redirectUrl == null)
+            {
+                redirectUrl = Url.Action("Default", "BackOffice");
```

Comment:
```suggestion
                redirectUrl = Url.Action(nameof(Default), this.GetControllerName());
```

Same goes for the rest of the file